src/lib/util.js

```diff
@@ -27,13 +28,13 @@ module.exports = {
         return hudScriptElement.getAttribute('data-request-id');
     },
     resolveClientUrl: function(requestId, follow) {
-        var clientTemplate = hudScriptElement.getAttribute('data-client-template');
+        var clientTemplate = new UriTemplate(hudScriptElement.getAttribute('data-client-template'));
 
-        var params = '&requestId=' + requestId;
-        params += '&metadataUri=' + encodeURIComponent(hudScriptElement.getAttribute('data-metadata-template')); // This happens to be fully resolved already
-        params += follow ? '&follow=true' : '';
-
-        return clientTemplate.replace('{&requestId,follow,metadataUri}', params); // TODO: This should probably be resolved with a URI Template library
+        return clientTemplate.fill({
+            requestId: requestId,
+            follow: !!follow,
```

Comment:
Follow should be set to "true" in this context i.e. when ever the user opens Glimpse from HUD, we should be automatically in follow mode... this token lets the client know that this is the case.